xbmc/cores/RetroPlayer/savestates/SavestateDatabase.cpp

```diff
@@ -152,7 +152,7 @@ void CSavestateDatabase::GetSavestateItem(const ISavestate& savestate,
                                           const std::string& savestatePath,
                                           CFileItem& item)
 {
-  CDateTime dateUTC = CDateTime::FromUTCDateTime(savestate.Created());
+  CDateTime dateUTC = savestate.Created();
```

Comment:
I'm checking and this is the only line of RetroPlayer code that's changed, is this change because the extra function call wasn't needed in the first place?

Reply:
It is because internal CDateTime representation was local time, but I have changed it to UTC (as per IANA's guidelines and common sense) so the call became obsolete.

Reply:
Glad to hear internal moved to UTC!

Reply:
Your comment made me check how creation timestamp is serialized inside the savestate and it appears that `CDateTime::GetFromW3CDateTime()` is used inside a FlatBuffers [container](https://github.com/xbmc/xbmc/blob/5a05aa2c4e4ae7f416803ae23efbc725840429e2/xbmc/cores/RetroPlayer/savestates/SavestateFlatBuffer.cpp#L288). This is actually pretty wise design choice as it mitigates the effects of internal representation changes inside CDateTime :)

However, this [line](https://github.com/xbmc/xbmc/blob/5a05aa2c4e4ae7f416803ae23efbc725840429e2/xbmc/cores/RetroPlayer/savestates/SavestateFlatBuffer.cpp#L204) implies a direct savestate is (was?) possible. Now all savestates are FlatBuffers-based, correct?

Reply:
[This](https://github.com/xbmc/xbmc/blob/5a05aa2c4e4ae7f416803ae23efbc725840429e2/xbmc/cores/RetroPlayer/savestates/SavestateFlatBuffer.cpp#L621) and [this](https://github.com/xbmc/xbmc/blob/5a05aa2c4e4ae7f416803ae23efbc725840429e2/xbmc/cores/RetroPlayer/savestates/SavestateDatabase.cpp#L61) suggest no direct savestates possible now :) Sweet!